pygmt/tests/test_datasets_earth_day.py

```diff
@@ -0,0 +1,44 @@
+"""
+Test basic functionality for loading Blue Marble datasets.
+"""
+
+import numpy as np
+import numpy.testing as npt
+import pytest
+from pygmt.datasets import load_blue_marble
+
+rioxarray = pytest.importorskip("rioxarray")
+
```

Comment:
This one can be removed, right?
```suggestion
```

Perhaps we should check if CRS exists when rioxarray is installed?